### ISSUE TITLE
Freeze when someone is typing

### ISSUE THREAD

**Issue by edricus** [diamondburned/6cord#1]
Hello
OS: Arch, Linux 5.0
Terminal: Termite (tried with xterm, same)
Go: go-2:1.12-1
I exec the bin created with the command go get -u
Bug: When I go on a active channel, I see the [username] is typing... but the message is already there when I check with my phone and when I try to go on a other channel, 6cord freeze, have to close it. When I send msg in the channel it work fine.
Sorry I don't know how to use the -d option i'm not familiar with GO
Might this help you, I love this app

**Comment by diamondburned** [comment]
lmao I was poking around with this yesterday but was too tired so I went to sleep. [I've opened the issue on source](https://gitlab.com/diamondburned/6cord/issues/2) but I haven't got time around it yet, I'll have a look again for the next few hours. 

Side note, I pushed this without really thinking people would notice them. [Turns out they do...](http://u.cubeupload.com/diamondburned/h6rUT5.png)
